extensions/quick-surf/package.json

```
@@ -0,0 +1,98 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quick-surf",
+  "title": "Quick Surf",
+  "description": "Quick Surf between browsers or email clients.",
+  "icon": "quick-surf.png",
+  "author": "koinzhang",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "quick-surf",
+      "title": "Surf with...",
```

```suggestion
      "title": "Quick Surf",
```
Maybe change title to this?